workflow-defects.md

```diff
@@ -0,0 +1,59 @@
+## Defects
+
+A **bug report is like a cake**, if you **don't take the time** to bake it well enough, your guests will have a **long and hard time digesting it**. If you **bake it for too long**, then the whole kitchen is going to **smell like burnt caramel**. 
+
+A defect is fixed as fast as it is well–communicated (sufficient information, well written).
+If the context and the way to reproduce a defect are made clear (not too much information, not too less), it will be easier for everyone to implement it, test it, refer to it…
+
+---
+
+Quick links: [Summary](#summary) | [Template](#template)
+
+---
+
+### Summary
+
+The summary of a defect should briefly describe the problem.
+Ideally, it would mention the type of user, the topic and the location of the problem. These are all valid summaries:
+
+- “I can’t upload a picture on the add property page”
+- “Edit individual profile is not loading”
+- “As a consumer, I can’t add a product to cart on the recommendation page”
+- “I can’t add a promotion code on checkout page”
+
+### Template
+
+We’re using the following template to create defects. It’s using [markdown][2].
+You should get familiar with it if you’re not already, **it’s awesome!**
+
+1. The beginning part of the description explains how to reproduce the defect.
+2. “Acceptance Criteria” explains the expected workflow.
+
+```markdown
+1. [If I do A.]
+1. [B doesn’t happen.]
+
+### Acceptance Criteria
```

Comment:
I don't like the idea of having acceptance criteria in a defect. It can lead to people adding new behavior there that should have gone in a user story. Can we call it something else? Like `Expected Behavior`. Ideally it should refer to an implemented story where the acceptance criteria was defined.